be/tool/beck.c

```diff
@@ -616,10 +618,12 @@ int main(int argc, char **argv)
 		   M0_STRINGARG('a', "stob domain path",
 			   LAMBDA(void, (const char *s) {
 				   beck_builder.b_stob_path = s;
+				   b_stob_path_length = sizeof(s);
```

Comment:
This is wrong.
The `sizeof(s)` equals to `sizeof(const char *s)`, and this is size of a pointer.

Reply:
Yes got it thanks for reviewing, will do the changes.